addon/src/components/animated-container.hbs

```diff
@@ -1,4 +1,4 @@
-{{#if this.useElementHelper}}
+{{#if (macroDependencySatisfies 'ember-element-helper' '^0.6.1')}}
```

Comment:
@mike-engel how would this work for `ember-element-helper` higher versions like v0.7 and v0.8?

Reply:
build still fails with the same error like #455 (and #633 which is the same thing)

```
Error: Attempted to resolve `macroDependencySatisfies`, which was expected to be a helper, but nothing was found.
```

Reply:
> @mike-engel how would this work for ember-element-helper higher versions like v0.7 and v0.8?

Thanks for catching this. This was a lazy copy/paste from the getter 😅

Reply:
@SergeAstapov I wonder if this is a bug or just expected behavior? [This matrix](https://github.com/embroider-build/embroider/issues/397) is out of date (I think), but `macroDependencySatisfies` has no runtime function. Additionally, the docs explicitly say that dead branches are only removed on production builds:

> The macroCondition macro allows branch level code isolation (and deletion in the case of production builds)

So either there's a bug where ember classic builds (although, this is still happening on my embroider build, too) aren't actually replacing code, or this is as expected. I'd assume the former, but I've no idea how to test that theory.

Reply:
Huh, I guess macros without a runtime should always be pruning branches: https://github.com/embroider-build/embroider/pull/1468

Reply:
My (maybe naive) fix for templates: https://github.com/embroider-build/embroider/pull/1688

Reply:
Additionally, I think `macroCondition (macroDependencySatisfies 'ember-element-helper' '>=0.6.1')` should also work if that PR doesn't get accepted